ios/FluentUI/People Picker/Avatar.swift

```diff
@@ -16,10 +16,16 @@ public protocol Avatar {
     var secondaryText: String { get }
     var image: UIImage? { get }
 
+    /// The color that represents this avatar.
+    /// This color will override the initials view's background color.
```

Comment:
We don't need to worry about _avatarBackgroundColor_. The API is actually logic as it is, let me explain.
If you call @objc public func setup(avatar: Avatar?), it will setup the AvatarView with the information from the object that conforms to the Avatar protocol. This could include the avatar's color.
_avatarBackgroundColor_  on the other hand is a property that can be used to manually set the avatar's background color. _avatarBackgroundColor_ is more of an override property.

You could call setup(avatar) and this will set the AvatarView's _avatarBackgroundColor_ to the avatar's color. Then, if you want to override this, you can use the _avatarBackgroundColor_ property. Works out great!